terraform/api.tf

```diff
@@ -214,6 +214,7 @@ resource "aws_cloudfront_distribution" "univaf_api" {
     max_ttl                = 3600
 
     forwarded_values {
+      headers = ["Host", "Origin"]
```

Comment:
If I understand the docs right, this will both forward these headers and include them in the cache key. I think that’s reasonable for these, but if we have others we only want to forward in the future (e.g. `User-Agent`), I think we have to create separate [`aws_cloudfront_origin_request_policy`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_origin_request_policy) and [`aws_cloudfront_cache_policy`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_cache_policy) resources.